manager/integration/tests/test_rwx.py

```diff
@@ -572,4 +575,78 @@ def test_rwx_offline_expansion(): # NOQA
     And
     - 1.5 Gi of data is successfully written to the expanded volume
     """
-    pass
+    pvc_name = 'pvc-deployment-rwx-expand-test'
+    pvc['metadata']['name'] = pvc_name
+    pvc['spec']['storageClassName'] = 'longhorn'
+    pvc['spec']['accessModes'] = ['ReadWriteMany']
+    pvc['spec']['resources']['requests']['storage'] = str(1 * Gi)
+
+    core_api.create_namespaced_persistent_volume_claim(
+        body=pvc, namespace='default')
+
+    deployment = make_deployment_with_pvc(
+        'deployment-rwx-expand-test', pvc_name, replicas=1)
+    apps_api = get_apps_api_client()
+    create_and_wait_deployment(apps_api, deployment)
+
+    pv_name = get_volume_name(core_api, pvc_name)
+    share_manager_name = 'share-manager-' + pv_name
+    deployment_label_selector = "name=" + \
+                                deployment["metadata"]["labels"]["name"]
+
+    deployment_pod_list = \
+        core_api.list_namespaced_pod(namespace="default",
+                                     label_selector=deployment_label_selector)
+
+    assert deployment_pod_list.items.__len__() == 1
```

Comment:
What's the purpose to check the number of the deployment pod? Isn't it quite straightforward? why need to validate it? This is a trivial assertion to check here, so I would suggest just assert what we need to assert (related to the test case) and just let it fail if there is no pod created for the deployment (it's about K8s not Longhorn).  

Below is good instead of verifying the number of pod in the deployment..

```python
    deployment_pod_list = \
        core_api.list_namespaced_pod(namespace="default",
                                     label_selector=deployment_label_selector)
    pod_name = deployment_pod_list.items[0].metadata.name
```

Reply:
I removed redundant code, thank you